### PR TITLE
feat: change mirror node and explorer from subchart to individual chart

### LINKED ISSUE #617: Update mirror node deployment to deploy mirror node external of Solo Charts, similar to JSON RPC Relay
- mirror node deployment using chart directly instead of leveraging solo-deployment sub-chart settings
- mirror node explorer deployment using chart directly instead of leveraging solo-deployment sub-chart settings

---
- relates to: https://github.com/hashgraph/full-stack-testing/issues/974
- depends on: https://github.com/hashgraph/full-stack-testing/issues/675
- depends on: https://github.com/hashgraph/hedera-mirror-node-explorer/issues/877

### PR DESCRIPTION
**Pull request by JeffreyDallas**
## Description

This pull request changes the following:

* change mirror node and explorer from subchart to individual chart

### Related Issues

* Closes #617 

---

### Notes

Mus not use `hedera-mirror` as release name, since
k8 would auto generate env variable `${SERVICE_NAME}_PORT`
which lead to a new `HEDERA_MIRROR_REST_PORT` clash with pre-existing env variable
used by mirror node.

https://stackoverflow.com/questions/45759205/what-environment-variables-are-created-in-kubernetes-by-default

image

### CODE REVIEW

File: resources/hedera-explorer-values.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+ingress:
```
Comment (jeromy-cannon) [inline resources/hedera-explorer-values.yaml:1]: ingress might require `haproxyIngressController`, let me know if that is the case or if I'm mistaken.

File: src/commands/flags.ts [hunk 2]
```diff
@@ -323,7 +323,7 @@ export const enableHederaExplorerTls: CommandFlag = {
   constName: 'enableHederaExplorerTls',
   name: 'enable-hedera-explorer-tls',
   definition: {
-    describe: 'Enable the Hedera Explorer TLS, defaults to false',
+    describe: 'Enable the Hedera Explorer TLS, defaults to false, must also enable deployCertManager and deployCertManagerCrds',
```
Comment (jeromy-cannon) [inline src/commands/flags.ts:326]:
```suggestion
    describe: 'Enable the Hedera Explorer TLS, defaults to false, requires certManager and certManagerCrds, which can be deployed through solo-cluster-setup chart or standalone',
```

**Review by jeromy-cannon — APPROVED** [review]

File: resources/hedera-explorer-values.yaml (hunk 1 above)
Comment (jeromy-cannon) [reply resources/hedera-explorer-values.yaml:1]: @JeffreyDallas

Comment (JeffreyDallas) [reply resources/hedera-explorer-values.yaml:1]: I think this ingress is different one vs haproxy-ingress

Comment (jeromy-cannon) [reply resources/hedera-explorer-values.yaml:1]: haproxyIngressController is what we used to configure the ingress in hedera mirror node explorer, those ingress value configurations are used here:

https://github.com/hashgraph/hedera-mirror-node-explorer/blob/main/chart/templates/ingress.yaml

As long as it tests all okay we should be good.  self-signed can be tested locally.  I had previously used GKE to test the letsencrypt signed certificates.

Comment (JeffreyDallas) [reply resources/hedera-explorer-values.yaml:1]: Test passed with new explorer chart 0.2.1 
with both `selfSignedClusterIssuer` enabled or disabled

**Pull request merged**